docs/source/locales/de/LC_MESSAGES/contribute.po

```diff
@@ -0,0 +1,49 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2019, High Fidelity. © Copyright 2020, Vircadia.
+# This file is distributed under the same license as the Vircadia Docs
+# package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2020.
+#
+#, fuzzy
```

Comment:
"fuzzy"?

Reply:
This is used by the translation softwares to mark strings that need a look at. Either the source string has been changed, or there is not a proper translation yet.